Google Resolver can not run with OneSignal  2.7.5, Firebase 5.2.1, & play-services-resolver-1.2.88.0+

**Status from @jkasten2:**
Issue is is due to Google's `unity-jar-resolver`, a ticket has been opened under googlesamples/unity-jar-resolver#149 to fix this issue.
Workaround fix in PR #152, this will be included in the next release.

**Orignal message:**
Hi,
Please help, I am having a problem when trying to Force resolve 

One signal 2.7.5
Unity 2017.4.4f1
Firebase Analytics 5.2.1
play-services-resolver-1.2.89.0
Android sdk update to max
Java sdk 1.8


When trying to Force Resolve, We have the problem
Without One signal, the Resolve works well.
Thank so much


Gradle failed to fetch dependencies.

Failed to run '/Users/imac/Downloads/1943RevolutionTeam/Temp/PlayServicesResolverGradle/gradlew -b "/Users/imac/Downloads/1943RevolutionTeam/Temp/PlayServicesResolverGradle/PlayServicesResolver.scripts.download_artifacts.gradle" --no-daemon "-PANDROID_HOME=/Users/imac/Library/Android/sdk" "-PTARGET_DIR=/Users/imac/Downloads/1943RevolutionTeam/Assets/Plugins/Android" "-PMAVEN_REPOS=file:////Users/imac/Downloads/1943RevolutionTeam/Assets/Firebase/m2repository" "-PPACKAGES_TO_COPY=com.android.support:support-v4:25.3.1;com.android.support:appcompat-v7:25.3.1;com.android.support:cardview-v7:25.3.1;com.android.support:customtabs:25.3.1;com.google.firebase:firebase-analytics:16.0.1;com.google.firebase:firebase-core:16.0.1;com.google.firebase:firebase-iid:[17.0.0];com.google.firebase:firebase-analytics-unity:5.2.1;com.google.android.gms:play-services-base:15.0.1;com.google.firebase:firebase-common:16.0.1;com.google.firebase:firebase-app-unity:5.2.1;com.google.firebase:firebase-messaging:17.3.0;com.google.firebase:firebase-messaging-unity:5.2.1;com.google.android.gms:play-services-base:[10.2.1, 12.1.0);com.google.firebase:firebase-messaging:[10.2.1, 12.1.0);com.google.android.gms:play-services-location:[10.2.1, 12.1.0);com.android.support:support-v4:[26.0.0, 27.2.0);com.android.support:customtabs:[26.0.0, 27.2.0)"'
stdout:

> Configure project :
ANDROID_HOME: /Users/imac/Library/Android/sdk
MAVEN_REPOS: name=maven url=file:/Users/imac/Downloads/1943RevolutionTeam/Assets/Firebase/m2repository/
MAVEN_REPOS: name=maven2 url=file:/Users/imac/Library/Android/sdk/extras/android/m2repository/
MAVEN_REPOS: name=maven3 url=file:/Users/imac/Library/Android/sdk/extras/google/m2repository/
MAVEN_REPOS: name=maven4 url=https://maven.google.com
MAVEN_REPOS: name=MavenLocal url=file:/Users/imac/.m2/repository
MAVEN_REPOS: name=BintrayJCenter url=https://jcenter.bintray.com/
MAVEN_REPOS: name=MavenRepo url=https://repo.maven.apache.org/maven2/
PACKAGES_TO_COPY: com.android.support:support-v4:25.3.1
PACKAGES_TO_COPY: com.android.support:appcompat-v7:25.3.1
PACKAGES_TO_COPY: com.android.support:cardview-v7:25.3.1
PACKAGES_TO_COPY: com.android.support:customtabs:25.3.1
PACKAGES_TO_COPY: com.google.firebase:firebase-analytics:16.0.1
PACKAGES_TO_COPY: com.google.firebase:firebase-core:16.0.1
PACKAGES_TO_COPY: com.google.firebase:firebase-iid:[17.0.0]
PACKAGES_TO_COPY: com.google.firebase:firebase-analytics-unity:5.2.1
PACKAGES_TO_COPY: com.google.android.gms:play-services-base:15.0.1
PACKAGES_TO_COPY: com.google.firebase:firebase-common:16.0.1
PACKAGES_TO_COPY: com.google.firebase:firebase-app-unity:5.2.1
PACKAGES_TO_COPY: com.google.firebase:firebase-messaging:17.3.0
PACKAGES_TO_COPY: com.google.firebase:firebase-messaging-unity:5.2.1
PACKAGES_TO_COPY: com.google.android.gms:play-services-base:[10.2.1, 12.1.0]
PACKAGES_TO_COPY: com.google.firebase:firebase-messaging:[10.2.1, 12.1.0]
PACKAGES_TO_COPY: com.google.android.gms:play-services-location:[10.2.1, 12.1.0]
PACKAGES_TO_COPY: com.android.support:support-v4:[26.0.0, 27.2.0]
PACKAGES_TO_COPY: com.android.support:customtabs:[26.0.0, 27.2.0]
TARGET_DIR: /Users/imac/Downloads/1943RevolutionTeam/Assets/Plugins/Android
Resolution attempt 1: packages [com.google.firebase:firebase-common:16.0.1, com.android.support:appcompat-v7:25.3.1, com.google.android.gms:play-services-location:[10.2.1, 12.1.0], com.google.firebase:firebase-core:16.0.1, com.google.android.gms:play-services-base:15.0.1, com.android.support:support-v4:25.3.1, com.android.support:customtabs:25.3.1, com.google.firebase:firebase-analytics:16.0.1, com.google.firebase:firebase-analytics-unity:5.2.1@srcaar, com.google.firebase:firebase-app-unity:5.2.1@srcaar, com.google.firebase:firebase-messaging-unity:5.2.1@srcaar, com.google.firebase:firebase-iid:[17.0.0], com.google.firebase:firebase-messaging:17.3.0, com.android.support:cardview-v7:25.3.1]
com.google.android.gms:play-services-ads-identifier conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1
com.google.android.gms:play-services-base conflicting due to package(s):
- com.google.android.gms:play-services-base:15.0.1
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-base:12.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
com.google.android.gms:play-services-basement conflicting due to package(s):
- com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-base:12.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-base:12.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-basement:12.0.1
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-tasks:12.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1/com.google.android.gms:play-services-basement:[15.0.1,16.0.0]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1/com.google.android.gms:play-services-basement:[15.0.1,16.0.0]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1/com.google.android.gms:play-services-basement:[15.0.1,16.0.0]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1/com.google.android.gms:play-services-basement:[15.0.1,16.0.0]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-measurement-connector:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-measurement-connector:16.0.0/com.google.android.gms:play-services-basement:15.0.1
com.google.android.gms:play-services-stats conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-stats:15.0.1
com.google.android.gms:play-services-tasks conflicting due to package(s):
- com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-base:12.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-tasks:12.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
com.google.firebase:firebase-analytics conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]
com.google.firebase:firebase-analytics-impl conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]
com.google.firebase:firebase-common conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1
com.google.firebase:firebase-iid-interop conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0
com.google.firebase:firebase-measurement-connector conflicting due to package(s):
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-measurement-connector:16.0.0
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-measurement-connector:16.0.0
Resolution attempt: 1, conflicts detected: true, updated packages: true
Resolution attempt 2: packages [com.android.support:customtabs:26.1.0, com.google.firebase:firebase-common:16.0.1+, com.google.android.gms:play-services-base:15.0.1+, com.android.support:cardview-v7:26.1.0, com.android.support:support-media-compat:26.1.0, com.android.support:support-vector-drawable:26.1.0, com.google.android.gms:play-services-location: 12.1.0+, com.android.support:support-fragment:26.1.0, com.android.support:animated-vector-drawable:26.1.0, com.android.support:support-annotations:26.1.0, com.google.firebase:firebase-core:16.0.1+, com.google.firebase:firebase-analytics-unity:5.2.1@srcaar, com.google.firebase:firebase-messaging:17.3.0+, com.android.support:support-core-utils:26.1.0, com.android.support:appcompat-v7:26.1.0, com.android.support:support-compat:26.1.0, com.android.support:support-v4:26.1.0, com.google.firebase:firebase-app-unity:5.2.1@srcaar, com.google.firebase:firebase-messaging-unity:5.2.1@srcaar, com.google.firebase:firebase-analytics:16.0.1+, com.google.firebase:firebase-iid:[17.0.0], com.android.support:support-core-ui:26.1.0]
Resolve failed due to com.google.android.gms:play-services-location: 12.1.0+ missing for project :, attempted to use com.google.android.gms:play-services-location: 12.1.0+, failed due to org.gradle.internal.resolve.ModuleVersionNotFoundException: Could not find any version that matches com.google.android.gms:play-services-location: 12.1.0+.
Versions that do not match:
  - 11.0.4
  - 11.0.2
  - 11.0.1
  - 11.0.0
  - 10.2.6
  - + 33 more
Searched in the following locations:
  - https://maven.google.com/com/google/android/gms/play-services-location/maven-metadata.xml
  - https://jcenter.bintray.com/com/google/android/gms/play-services-location/maven-metadata.xml
  - https://jcenter.bintray.com/com/google/android/gms/play-services-location/
  - https://repo.maven.apache.org/maven2/com/google/android/gms/play-services-location/maven-metadata.xml
  - https://repo.maven.apache.org/maven2/com/google/android/gms/play-services-location/

stderr:

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/imac/Downloads/1943RevolutionTeam/Temp/PlayServicesResolverGradle/PlayServicesResolver.scripts.download_artifacts.gradle' line: 745

* What went wrong:
A problem occurred evaluating root project 'PlayServicesResolverGradle'.
> For input string: " 12"

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 12s

exit code: 1

Here is another log trace
I try to make a small and clean project. It only has Firebase analytics, firebase messaging and one signal


Refresh: detecting if any assets need to be imported or removed ... 
Refresh: elapses 0.010980 seconds (Nothing changed)
Gradle failed to fetch dependencies.

Failed to run '/Users/imac/bitbucket/CleanBuild/Temp/PlayServicesResolverGradle/gradlew -b "/Users/imac/bitbucket/CleanBuild/Temp/PlayServicesResolverGradle/PlayServicesResolver.scripts.download_artifacts.gradle" --no-daemon "-PANDROID_HOME=/Users/imac/Library/Android/sdk" "-PTARGET_DIR=/Users/imac/bitbucket/CleanBuild/Assets/Plugins/Android" "-PMAVEN_REPOS=file:////Users/imac/bitbucket/CleanBuild/Assets/Firebase/m2repository" "-PPACKAGES_TO_COPY=com.google.firebase:firebase-analytics:16.0.1;com.google.firebase:firebase-core:16.0.1;com.google.firebase:firebase-iid:[17.0.0];com.google.firebase:firebase-analytics-unity:5.2.1;com.google.android.gms:play-services-base:15.0.1;com.google.firebase:firebase-common:16.0.1;com.google.firebase:firebase-app-unity:5.2.1;com.google.firebase:firebase-messaging:17.3.0;com.google.firebase:firebase-messaging-unity:5.2.1;com.google.android.gms:play-services-base:[10.2.1, 12.1.0);com.google.firebase:firebase-messaging:[10.2.1, 12.1.0);com.google.android.gms:play-services-location:[10.2.1, 12.1.0);com.android.support:support-v4:[26.0.0, 27.2.0);com.android.support:customtabs:[26.0.0, 27.2.0)"'
stdout:

> Configure project :
ANDROID_HOME: /Users/imac/Library/Android/sdk
MAVEN_REPOS: name=maven url=file:/Users/imac/bitbucket/CleanBuild/Assets/Firebase/m2repository/
MAVEN_REPOS: name=maven2 url=file:/Users/imac/Library/Android/sdk/extras/android/m2repository/
MAVEN_REPOS: name=maven3 url=file:/Users/imac/Library/Android/sdk/extras/google/m2repository/
MAVEN_REPOS: name=maven4 url=https://maven.google.com
MAVEN_REPOS: name=MavenLocal url=file:/Users/imac/.m2/repository
MAVEN_REPOS: name=BintrayJCenter url=https://jcenter.bintray.com/
MAVEN_REPOS: name=MavenRepo url=https://repo.maven.apache.org/maven2/
PACKAGES_TO_COPY: com.google.firebase:firebase-analytics:16.0.1
PACKAGES_TO_COPY: com.google.firebase:firebase-core:16.0.1
PACKAGES_TO_COPY: com.google.firebase:firebase-iid:[17.0.0]
PACKAGES_TO_COPY: com.google.firebase:firebase-analytics-unity:5.2.1
PACKAGES_TO_COPY: com.google.android.gms:play-services-base:15.0.1
PACKAGES_TO_COPY: com.google.firebase:firebase-common:16.0.1
PACKAGES_TO_COPY: com.google.firebase:firebase-app-unity:5.2.1
PACKAGES_TO_COPY: com.google.firebase:firebase-messaging:17.3.0
PACKAGES_TO_COPY: com.google.firebase:firebase-messaging-unity:5.2.1
PACKAGES_TO_COPY: com.google.android.gms:play-services-base:[10.2.1, 12.1.0]
PACKAGES_TO_COPY: com.google.firebase:firebase-messaging:[10.2.1, 12.1.0]
PACKAGES_TO_COPY: com.google.android.gms:play-services-location:[10.2.1, 12.1.0]
PACKAGES_TO_COPY: com.android.support:support-v4:[26.0.0, 27.2.0]
PACKAGES_TO_COPY: com.android.support:customtabs:[26.0.0, 27.2.0]
TARGET_DIR: /Users/imac/bitbucket/CleanBuild/Assets/Plugins/Android
Resolution attempt 1: packages [com.google.firebase:firebase-common:16.0.1, com.google.firebase:firebase-analytics-unity:5.2.1@srcaar, com.google.android.gms:play-services-location:[10.2.1, 12.1.0], com.google.firebase:firebase-core:16.0.1, com.google.firebase:firebase-app-unity:5.2.1@srcaar, com.google.firebase:firebase-messaging-unity:5.2.1@srcaar, com.google.android.gms:play-services-base:15.0.1, com.google.firebase:firebase-iid:[17.0.0], com.google.firebase:firebase-messaging:17.3.0, com.android.support:customtabs:[26.0.0, 27.2.0], com.google.firebase:firebase-analytics:16.0.1, com.android.support:support-v4:[26.0.0, 27.2.0]]
com.google.android.gms:play-services-ads-identifier conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1
com.google.android.gms:play-services-base conflicting due to package(s):
- com.google.android.gms:play-services-base:15.0.1
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-base:12.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1
com.google.android.gms:play-services-basement conflicting due to package(s):
- com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-base:12.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-base:12.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-basement:12.0.1
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-tasks:12.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1/com.google.android.gms:play-services-basement:[15.0.1,16.0.0]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1/com.google.android.gms:play-services-basement:[15.0.1,16.0.0]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1/com.google.android.gms:play-services-basement:[15.0.1,16.0.0]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-ads-identifier:15.0.1/com.google.android.gms:play-services-basement:[15.0.1,16.0.0]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-measurement-connector:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-stats:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]/com.google.android.gms:play-services-basement:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-basement:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-measurement-connector:16.0.0/com.google.android.gms:play-services-basement:15.0.1
com.google.android.gms:play-services-stats conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-stats:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-stats:15.0.1
com.google.android.gms:play-services-tasks conflicting due to package(s):
- com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-base:12.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.android.gms:play-services-location:[10.2.1, 12.1.0]/com.google.android.gms:play-services-tasks:12.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-common:16.0.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
- com.google.firebase:firebase-messaging:17.3.0/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1/com.google.android.gms:play-services-tasks:15.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0/com.google.android.gms:play-services-base:15.0.1/com.google.android.gms:play-services-tasks:[15.0.1]
com.google.firebase:firebase-analytics conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]
com.google.firebase:firebase-analytics-impl conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]
com.google.firebase:firebase-common conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-common:16.0.0
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-common:16.0.1
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-common:16.0.1
com.google.firebase:firebase-iid-interop conflicting due to package(s):
- com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-analytics:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-analytics:[16.0.1]/com.google.firebase:firebase-analytics-impl:[16.1.1]/com.google.firebase:firebase-iid:16.0.0/com.google.firebase:firebase-iid-interop:16.0.0
- com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-iid:[17.0.0]/com.google.firebase:firebase-iid-interop:16.0.0
com.google.firebase:firebase-measurement-connector conflicting due to package(s):
- com.google.firebase:firebase-core:16.0.1/com.google.firebase:firebase-measurement-connector-impl:16.0.1/com.google.firebase:firebase-measurement-connector:16.0.0
- com.google.firebase:firebase-messaging:17.3.0/com.google.firebase:firebase-measurement-connector:16.0.0
Resolution attempt: 1, conflicts detected: true, updated packages: true
Resolution attempt 2: packages [com.google.firebase:firebase-common:16.0.1+, com.android.support:support-v4:27.1.1, com.google.android.gms:play-services-base:15.0.1+, com.android.support:support-compat:27.1.1, com.android.support:support-core-ui:27.1.1, com.android.support:support-core-utils:27.1.1, com.android.support:support-media-compat:27.1.1, com.google.android.gms:play-services-location: 12.1.0+, com.google.firebase:firebase-core:16.0.1+, com.google.firebase:firebase-analytics-unity:5.2.1@srcaar, com.google.firebase:firebase-messaging:17.3.0+, com.google.firebase:firebase-app-unity:5.2.1@srcaar, com.google.firebase:firebase-messaging-unity:5.2.1@srcaar, com.android.support:customtabs:27.1.1, com.google.firebase:firebase-analytics:16.0.1+, com.google.firebase:firebase-iid:[17.0.0], com.android.support:support-fragment:27.1.1, com.android.support:support-annotations:27.1.1]
Resolve failed due to com.google.android.gms:play-services-location: 12.1.0+ missing for project :, attempted to use com.google.android.gms:play-services-location: 12.1.0+, failed due to org.gradle.internal.resolve.ModuleVersionNotFoundException: Could not find any version that matches com.google.android.gms:play-services-location: 12.1.0+.
Versions that do not match:
  - 11.0.4
  - 11.0.2
  - 11.0.1
  - 11.0.0
  - 10.2.6
  - + 33 more
Searched in the following locations:
  - https://maven.google.com/com/google/android/gms/play-services-location/maven-metadata.xml
  - https://jcenter.bintray.com/com/google/android/gms/play-services-location/maven-metadata.xml
  - https://jcenter.bintray.com/com/google/android/gms/play-services-location/
  - https://repo.maven.apache.org/maven2/com/google/android/gms/play-services-location/maven-metadata.xml
  - https://repo.maven.apache.org/maven2/com/google/android/gms/play-services-location/

stderr:

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/imac/bitbucket/CleanBuild/Temp/PlayServicesResolverGradle/PlayServicesResolver.scripts.download_artifacts.gradle' line: 745

* What went wrong:
A problem occurred evaluating root project 'PlayServicesResolverGradle'.
> For input string: " 12"

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 13s

exit code: 1

UnityEngine.DebugLogHandler:Internal_Log(LogType, String, Object)
UnityEngine.DebugLogHandler:LogFormat(LogType, Object, String, Object[])
UnityEngine.Logger:Log(LogType, Object)
UnityEngine.Debug:LogError(Object)
Google.Logger:Log(String, LogLevel)
GooglePlayServices.PlayServicesResolver:Log(String, LogLevel)
GooglePlayServices.<GradleResolution>c__AnonStorey17:<>m__27(Result)
GooglePlayServices.<GradleResolution>c__AnonStorey19:<>m__32()
Google.RunOnMainThread:ExecuteNext()
Google.RunOnMainThread:<ExecuteAll>m__6()
Google.RunOnMainThread:RunAction(Action)
Google.RunOnMainThread:ExecuteAll()
Google.RunOnMainThread:Run(Action, Boolean)
GooglePlayServices.<GradleResolution>c__AnonStorey17:<>m__28(Result)
GooglePlayServices.<GradleResolution>c__AnonStorey17:<>m__29(Result)
GooglePlayServices.ProgressReporter:SignalComplete()
GooglePlayServices.ProgressReporter:CommandLineToolCompletion(Result)
GooglePlayServices.<RunAsync>c__AnonStorey2:<>m__3()
Google.RunOnMainThread:ExecuteNext()
Google.RunOnMainThread:<ExecuteAll>m__6()
Google.RunOnMainThread:RunAction(Action)
Google.RunOnMainThread:ExecuteAll()
UnityEditor.EditorApplication:Internal_CallUpdateFunctions() (at /Users/builduser/buildslave/unity/build/Editor/Mono/EditorApplication.cs:127)
 
(Filename: /Users/builduser/buildslave/unity/build/Editor/Mono/EditorApplication.cs Line: 127)

Resolution failed

Failed to fetch the following dependencies:
com.google.firebase:firebase-analytics:16.0.1
com.google.firebase:firebase-core:16.0.1
com.google.firebase:firebase-iid:[17.0.0]
com.google.firebase:firebase-analytics-unity:5.2.1
com.google.android.gms:play-services-base:15.0.1
com.google.firebase:firebase-common:16.0.1
com.google.firebase:firebase-app-unity:5.2.1
com.google.firebase:firebase-messaging:17.3.0
com.google.firebase:firebase-messaging-unity:5.2.1
com.google.android.gms:play-services-base:[10.2.1, 12.1.0)
com.google.firebase:firebase-messaging:[10.2.1, 12.1.0)
com.google.android.gms:play-services-location:[10.2.1, 12.1.0)
com.android.support:support-v4:[26.0.0, 27.2.0)
com.android.support:customtabs:[26.0.0, 27.2.0)


UnityEngine.DebugLogHandler:Internal_Log(LogType, String, Object)
UnityEngine.DebugLogHandler:LogFormat(LogType, Object, String, Object[])
UnityEngine.Logger:Log(LogType, Object)
UnityEngine.Debug:LogError(Object)
Google.Logger:Log(String, LogLevel)
GooglePlayServices.PlayServicesResolver:Log(String, LogLevel)
GooglePlayServices.ResolverVer1_1:LogMissingDependenciesError(List`1)
GooglePlayServices.<DoResolutionUnsafe>c__AnonStorey1C:<>m__2E(List`1)
GooglePlayServices.<GradleResolution>c__AnonStorey17:<>m__27(Result)
GooglePlayServices.<GradleResolution>c__AnonStorey19:<>m__32()
Google.RunOnMainThread:ExecuteNext()
Google.RunOnMainThread:<ExecuteAll>m__6()
Google.RunOnMainThread:RunAction(Action)
Google.RunOnMainThread:ExecuteAll()
Google.RunOnMainThread:Run(Action, Boolean)
GooglePlayServices.<GradleResolution>c__AnonStorey17:<>m__28(Result)
GooglePlayServices.<GradleResolution>c__AnonStorey17:<>m__29(Result)
GooglePlayServices.ProgressReporter:SignalComplete()
GooglePlayServices.ProgressReporter:CommandLineToolCompletion(Result)
GooglePlayServices.<RunAsync>c__AnonStorey2:<>m__3()
Google.RunOnMainThread:ExecuteNext()
Google.RunOnMainThread:<ExecuteAll>m__6()
Google.RunOnMainThread:RunAction(Action)
Google.RunOnMainThread:ExecuteAll()
UnityEditor.EditorApplication:Internal_CallUpdateFunctions() (at /Users/builduser/buildslave/unity/build/Editor/Mono/EditorApplication.cs:127)
 
(Filename: /Users/builduser/buildslave/unity/build/Editor/Mono/EditorApplication.cs Line: 127)

Refresh: detecting if any assets need to be imported or removed ... Refresh: elapses 0.011532 seconds (Nothing changed)

--- a/OneSignalExample/Assets/OneSignal/Editor/OneSignalDependencies.xml
+++ b/OneSignalExample/Assets/OneSignal/Editor/OneSignalDependencies.xml
@@ -1,9 +1,9 @@
 <dependencies>
   <androidPackages>
-    <androidPackage spec="com.google.android.gms:play-services-base:[10.2.1, 12.1.0)" />
-    <androidPackage spec="com.google.firebase:firebase-messaging:[10.2.1, 12.1.0)"/>
-    <androidPackage spec="com.google.android.gms:play-services-location:[10.2.1, 12.1.0)" />
-    <androidPackage spec="com.android.support:support-v4:[26.0.0, 27.2.0)" />
-    <androidPackage spec="com.android.support:customtabs:[26.0.0, 27.2.0)" />
+    <androidPackage spec="com.google.android.gms:play-services-base:[10.2.1, 12.1.0[" />
+    <androidPackage spec="com.google.firebase:firebase-messaging:[10.2.1, 12.1.0["/>
+    <androidPackage spec="com.google.android.gms:play-services-location:[10.2.1, 12.1.0[" />
+    <androidPackage spec="com.android.support:support-v4:[26.0.0, 27.2.0[" />
+    <androidPackage spec="com.android.support:customtabs:[26.0.0, 27.2.0[" />
   </androidPackages>
 </dependencies>

